doc/classes/VisualShader.xml

```diff
@@ -149,7 +149,7 @@
 		<member name="graph_offset" type="Vector2" setter="set_graph_offset" getter="get_graph_offset" default="Vector2(0, 0)">
 			The offset vector of the whole graph.
 		</member>
-		<member name="version" type="String" setter="set_version" getter="get_version" default="&quot;&quot;">
+		<member name="version" type="Dictionary" setter="set_version" getter="get_version" default="{}">
 		</member>
```

Comment:
```suggestion
		<member name="version" type="Dictionary" setter="set_version" getter="get_version" default="{}">
			The Godot version this [VisualShader] was designed for, in the form of a [Dictionary]. Example: [code]{"major": 4, "minor": 0}[/code]
			This is used by the editor to convert visual shaders from older Godot versions.
		</member>
```

Reply:
@Calinou Thanks, I've updated it and make you a co-author.